opentracing-spring-cloud/src/main/java/io/opentracing/contrib/spring/cloud/jms/JmsAutoConfiguration.java

```diff
@@ -35,33 +31,7 @@
 @ConditionalOnClass({Message.class, JmsTemplate.class})
 @ConditionalOnBean(Tracer.class)
 @ConditionalOnProperty(name = "opentracing.spring.cloud.jms.enabled", havingValue = "true", matchIfMissing = true)
+@Import(TracingJmsConfiguration.class)
 public class JmsAutoConfiguration {
 
-  @Bean
-  public JmsListenerAspect jmsListenerAspect() {
-    return new JmsListenerAspect();
-  }
-
-  @Bean
-  public JmsTemplate jmsTemplate(BeanFactory beanFactory, Tracer tracer) {
```

Comment:
I am wondering if this works when there are multiple `JmsTemplate` beans. @malafeev Could you write a simple test for it?

Reply:
I just moved it to java-jms. 
So behaviour should be the same.

Reply:
I see, but I am asking if you would be interested in writing a test for it.

Reply:
If I add 
```
    @Bean
    public JmsTemplate jmsTemplate() {
      return new JmsTemplate(connectionFactory());
    }
```
to `JmsTestConfiguration` then bean is not created but used `TracingJmsTemplate` from `TracingJmsConfiguration`

Reply:
should I commit that or create separate test class for that case?

Reply:
I am not sure what you mean exactly. Maybe a separate PR would be better.

Reply:
ok, I will create another PR for test